core/hotspot/cache/hash.go

```diff
@@ -0,0 +1,94 @@
+package cache
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/gob"
+	"hash/fnv"
+	"math"
+	"reflect"
+)
+
+func sum(k interface{}) uint64 {
```

Comment:
Could you please provide some benchmark for hash function?

Reply:
done